### DISCUSSION TITLE
How to use own icons in IconeButton

### DISCUSSION THREAD

**Discussion by abdokhalil07** [flet-dev/flet#4261] [Q&A] [answered]
### Question

i want to  add my icone to  IconButton control but it not work.
![icon_](https://github.com/user-attachments/assets/b937fd6e-6049-424b-9641-064deca04b25)

### Code sample

```python
def main(page: Page):

    Home_View = View(
        "home",
        appbar=AppBar(
            actions=[
                IconButton(icon=icons.SETTINGS_ETHERNET),
                IconButton(icon="assets/icons/log.png")
            ]
        )
    )

    def on_route(e):
        page.views.clear()
        page.views.append(Home_View)
        page.update()
        
    page.on_route_change = on_route
    page.go(page.route)
    page.update()

if __name__ == "__main__":
    app(main, assets_dir="assets")
```

### Error message

```bash
conButton must have either icon or a visible content specified.
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
Use [`IconButton.content`](https://flet.dev/docs/controls/iconbutton/#content) and set it to an `Image` with `src` being the path to your image `"/icons/log.png"`
